app/views/shop/_blocked_cookies.html.haml

```diff
@@ -0,0 +1,14 @@
+:javascript
+  $(document).ready(function() {
+    document.cookie = 'cookie_test=true';
```

Comment:
The javascript `document.cookie` doesn't overwrite other cookies:

https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie